.husky/pre-commit

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
-. "$(dirname $0)/_/husky.sh"
+. "$(dirname "$0")/_/husky.sh"
+
+# Skip pre-commit hook on CI
+[ -n "$CI" ] && exit 0
```

Comment:
I'm not sure if our GitHub Action workflow files correctly say they're running in a CI environment?

Reply:
Yes! Most CI services automatically expose a `CI` env var. You can see default ones for GitHub Actions here, including `CI` one: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Okay then! 😄